.vscode/settings.json

```diff
@@ -1,7 +1,15 @@
 {
-    "rust-analyzer.check.allTargets": false,
     "rust-analyzer.cargo.target": "x86_64-unknown-none",
-    "rust-analyzer.check.extraArgs": [
+    "rust-analyzer.check.ignore": [
+        "unexpected_cfgs"
```

Comment:
Might as well add `--check-cfg cfg(ktest)` and `--cfg ktest` to `"rust-analyzer.check.overrideCommand"`, which precisely suppressed the warning and allows checking for ktests.

Reply:
> Might as well add `--check-cfg cfg(ktest)` and `--cfg ktest` to `"rust-analyzer.check.overrideCommand"`, which precisely suppressed the warning and allows checking for ktests.

It seems that the `--check-cfg cfg(ktest)` and `--cfg ktest` require to modify the `RUSTFLAGS` environment variables. I use 
```json

    "rust-analyzer.check.extraEnv": {
        "RUSTFLAGS": "--check-cfg cfg(ktest) --cfg ktest"
    },

```
instead.

Reply:
@jellllly420 Are we supposed to use `cargo osdk check` rather than `cargo check`?

Reply:
> @jellllly420 Aren't we supposed to use `cargo osdk check` rather than `cargo check`?

The osdk just append rustflags:`--check-cfg cfg(ktest)` and pass the arguments to the `cargo check`. The command is similar to the `cargo check` if using `cargo osdk check`. I think it is ok to use `cargo check` here since osdk may change its usage in the future while `cargo check` is a stable one.

Reply:
> I'm afraid some small changes are necessary before enabling the check of ktests.
> 

I will fix it in this PR.